director/scripts/test/kinematic_test.py

```diff
@@ -56,19 +75,84 @@ def state_handler(self, channel, data):
         self.plant.SetPositions(self.context, msg.position)
         self.plant.SetVelocities(self.context, msg.velocity)
 
-        # TODO: pt_body and body_name are JSON options
-        # Once you can visualize, play around with these values to see what
-        # they do. Look in the .urdf file above to know valid body names
-        pt_body = np.array([0,0,0])
-        body_name = "toe_left"
-
-        # Use Drake's CalcPointsPositions to determine where that point is
-        # in the world
-        pt_world = self.plant.CalcPointsPositions(self.context,
-            self.plant.GetFrameByName(body_name), pt_body,
-            self.plant.world_frame())
-        print(pt_world.transpose())
-        #TODO instead of printing the position, visualize it
+        for data in self.data['data']:
```

Comment:
Parsing of the JSON file should occur once, before LCM handling starts. Everything needed should then be stored into convenient objects.